sql/pre_install/tables.sql

```diff
@@ -133,23 +133,26 @@ SELECT pg_catalog.pg_extension_config_dump(pg_get_serial_sequence('_timescaledb_
 -- the chunk's hypercube. Tuples that fall within the chunk's
 -- hypercube are stored in the chunk's data table, as given by
 -- 'schema_name' and 'table_name'.
+CREATE SEQUENCE IF NOT EXISTS _timescaledb_catalog.chunk_id_seq MINVALUE 1;
+
 CREATE TABLE IF NOT EXISTS _timescaledb_catalog.chunk (
-  id serial NOT NULL PRIMARY KEY,
+  id integer PRIMARY KEY DEFAULT nextval('_timescaledb_catalog.chunk_id_seq'),
   hypertable_id int NOT NULL REFERENCES _timescaledb_catalog.hypertable (id),
   schema_name name NOT NULL,
   table_name name NOT NULL,
   compressed_chunk_id integer REFERENCES _timescaledb_catalog.chunk (id),
   dropped boolean NOT NULL DEFAULT FALSE,
+  chunk_status smallint NOT NULL DEFAULT 0,
```

Comment:
Still think we should use an `INT` since there is nothing to gain from using a smaller type.

Reply:
I agree with this. It doesn't seem an extra 2 bytes per chunk would be a great overhead, and for future-proofing it would be simpler to have 32 potential flags. The reason is that I feel 16 are too few. There could be a lot of things we could do with those flags, e.g.
```
CHUNK_COMPRESSED
CHUNK_COMPRESSED_LZO
CHUNK_COMPRESSED_LZ4
CHUNK_COMPRESSED_BZIP2
...
CHUNK_DIRTY
CHUNK_INFLIGHT
CHUNK_UPTODATE
CHUNK_NEEDS_HEALING
CHUNK_ENCRYPTED
CHUNK_ENCRYPTED_AES128
CHUNK_ENCRYPTED_AES256
...
CHUNK_PRIMARY_REPLICA
CHUNK_LOCKED
CHUNK_ACTIVE
CHUNK_CHECKED
CHUNK_PINNED
```
Obviously, the above don't mean anything, but my point is that we could do a lot of things with this field.

Reply:
should this be called status instead since we dont prefix the other columns with chunk_

Reply:
renamed to status and changed type to int.